lib/ansible/plugins/action/copy.py

```diff
@@ -208,6 +208,13 @@ class ActionModule(ActionBase):
 
     TRANSFERS_FILES = True
 
+    def _ensure_invocation(self, result):
```

Comment:
bcoca mentioned a couple good points about this:

We should add a comment along the lines of:
```
# Note, adding invocation arguments here needs to be kept in sync with any no_log specified in the argument_spec in the module.  This is not automatic.
```

And we should also make sure that the content parameter doesn't end up in invocation (because content is marked as no_log in the copy argument spec).

You probably should use self._task.args.copy() so that we're not modifying the real task.args at that point too.

It will look something like:
``` python
if 'content' in result['invocation']:
    result['invocation'] = result['invocation'].copy()
    result['invocation']['content'] = 'CENSORED: content is a no_log parameter'
```

Reply:
@abadger I think I addressed them in the latest commit, but not on the exact line mentioned so GitHub doesn't quite "detect" that I addressed it. Please review when you have a moment.